### DISCUSSION TITLE
GitHub Repo Indexing

### DISCUSSION THREAD

**Discussion by phantomOPBro** [community/community#52404] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

What is the average timeframe for GitHub to index a repository that has been updated such that the updated repository will show up when a search term is specified for the new data that was updated in the repo?

**Comment by look** [comment, marked as answer]
We don't have an SLO for that, but typically it is a few minutes.

**Reply by phantomOPBro** [reply]
Look, is the indexing the same for newly created repos as opposed to repos that were just updated with modified or new files?

**Reply by look** [reply]
Good question. Right now, new repos don't get added as quickly. Try searching in the repository in question and it should get indexed after a few minutes.
